src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
Perhaps you can be more consistent in the spacing between your lines to improve readability as well!